File: 39-d-cubical-planet/planet.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    int x1, y1, z1;
    cin >> x1 >> y1 >> z1;

    int x2, y2, z2;
    cin >> x2 >> y2 >> z2;

    bool answer = pow(x1 - x2, 2) + pow(y1 - y2, 2) + pow(z1 - z2, 2) < 3;

    cout << (answer ? "YES" : "NO") << endl;
}

File: 39-d-cubical-planet/planet.py
x1, y1, z1 = map(int, input().split())
x2, y2, z2 = map(int, input().split())

answer = (x2 - x1) ** 2 + (y1 - y2) ** 2 + (z1 - z2) ** 2 < 3

print("YES" if answer else "NO")
